packages/replay/src/coreHandlers/handleDom.ts

```diff
@@ -47,13 +48,21 @@ function handleDom(handlerData: DomHandlerData): Breadcrumb | null {
     return null;
   }
 
+  const serializedNode = targetNode && ('__sn' in targetNode) ? targetNode.__sn  as elementNode & {id: number} : null;
   return createBreadcrumb({
     category: `ui.${handlerData.name}`,
     message: target,
     data: {
-      // Not sure why this errors, Node should be correct (Argument of type 'Node' is not assignable to parameter of type 'INode')
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      ...(targetNode ? { nodeId: record.mirror.getId(targetNode as any) } : {}),
+      ...(serializedNode ? {
+        nodeId: serializedNode.id,
```

Comment:
Should we remove this since its duplicated within the node object?

Reply:
UI relies on it so wanted to keep it here for now.

Reply:
👍